translate/canon/ListLanguageCodes.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright(c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START translate_list_codes]
+
+using Google.Cloud.Translation.V2;
```

Comment:
I suggest we add a comment that says "install Google.Cloud.Translation.V2 nuget package" or alternatively having a remark before embedded sample in the doc explaining what are the prerequisites for this to work.

Reply:
IIUC, the rubric wants the install command to appear in the product docs, not here.

Reply:
OK, sounds like remark before sample code per language in documentation will work, so no changes here.